packages/atomic/src/components/search/facets/atomic-automatic-facet-generator/atomic-automatic-facet-generator.tsx

```diff
@@ -18,15 +18,13 @@ import {Bindings} from '../../atomic-search-interface/atomic-search-interface';
 
 /**
  * The `atomic-automatic-facet-generator` is a component that renders the facets from
- * the automatic facets feature.
- *
- * Unlike traditional facets that need to be explicitly defined and requested in the query, automatic facets are dynamically generated by the index
+ * the automatic facets feature. Unlike regular facets that need to be explicitly defined
+ * and requested in the query, automatic facets are dynamically generated by the index
  * in response to the search query.
  *
- * It dynamically creates multiple `atomic-automatic-facet`
- * components based on the desiredCount prop.
+ * **Note:** This component renders multiple `atomic-automatic-facet` based on the desiredCount property.
  *
- * To read more about the automatic facet generator feature, see: https://docs.coveo.com/en/n9sd0159/
+ * To learn more about the automatic facet generator feature, see: [About the Facet Generator](https://docs.coveo.com/en/n9sd0159/).
  */
```

Comment:
@srosenberg-coveo hey sam, I changed also the description in the generator description. Do you have any comments? I am not sure about the 25th line.